interop-devnet/docker-compose.yml

```diff
@@ -352,3 +352,44 @@ services:
       # OP_BATCHER_DATA_AVAILABILITY_TYPE: blobs
     env_file:
       - "${PWD}/../.devnet-interop/env/l2/900201/op-batcher.env"
+
+  grafana:
+    image: grafana/grafana:11.1.0
+    restart: unless-stopped
+    env_file:
+      - monitoring/grafana.env
+    volumes:
+      - ./monitoring/grafana/provisioning/:/etc/grafana/provisioning/:ro
+      - ./monitoring/grafana/dashboards:/var/lib/grafana/dashboards
+      # - grafana_data:/var/lib/grafana
```

Comment:
Do we want to keep these commented-out docker volumes?

Reply:
I think it's nice, you might want to persist the grafana data or attach it to a volume. A little hint via comment how to do so is ok.